hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -362,6 +362,9 @@ private void setHmsTableParameters(String newMetadataLocation, Table tbl, Map<St
     // remove any props from HMS that are no longer present in Iceberg table props
     obsoleteProps.forEach(parameters::remove);
 
+    // remove any props that are controlled by HMS so it can handle them
+    parameters.remove(hive_metastoreConstants.DDL_TIME);
```

Comment:
Is this method called for `INSERT` as well? Maybe updating this every time is not entirely correct.

Reply:
Probably adding some test where we create a table and later insert to the table, and we make sure that the insert does not change the Ddl time

Reply:
Does `INSERT` not affect the DDL_TIME? I'm not as familiar with it, but that's surprising to me based on how I hear people using it.

Reply:
Theoretically it should not. DDL_TIME should be updated only with DDL operations.
Insert should not affect this, but if the alter table updating the statistics is incorrect, then it might.